### ISSUE TITLE
1.6.0 RC2 seems to work pretty well here :-)

### ISSUE THREAD

**Issue by newbie442** [cSploit/android#364]
1.6.0 RC2, Note 2, CM 11

Haven't tested it extensively, but it seems to work pretty well for me :-)

Initial installation does take a bit of time, and can appear to hang up during d/l install of various libraries. I'd guess this is a function of the device used, internet connection, other processes competing for resources, etc. 

There is no indication in the main window that the download/install is occurring, and impatient users may presume it has failed.

For me, when installing or upgrading, it is userful to shut down as many processes as possible; try to be patient; and frequently check the notification bar to see if subsequent D/l's are occurring

**Comment by newbie442** [comment]
On 10/04/2015 08:32 PM, tux-mind wrote:
>
> hi @newbie442 <https://github.com/newbie442> . I didn't get the point.
>
> what is the issue ? can you explain it better ?


Hi Tux-mind,

The point was to give you and the other devs some appreciation/positive 
feedback for all the work you're doing; also it was to indicate to any 
user casually passing through that you've made significant progress (on 
an important, imho, and complicated app) and are probably close to release.

Also it was to indicate to other folks in the forum the need to be a 
little patient before declaring an issue, as there is a lot going on.

As long as I have your ear, I put up another note suggesting that folks 
provide a little more information when posting - that it'll make your 
job a lot easier. I figured the message would be better accepted if it 
came from a fellow user, and not from one of you guys.

I figured you'd keep them up there for a while.....

At any rate,
Thanks For cSploit!!!!
